### PR TITLE
Devnet deploy build docker when needed

### PR DESCRIPTION
**Pull request by fkondej**

### CODE REVIEW

File: vars/pipelineDeploy.groovy [hunk 1]
```diff
@@ -116,7 +135,8 @@ void call() {
                         }
                         steps {
                             script {
-                                doGitClone('vega', params.VEGA_VERSION)
+                                def repoVars = doGitClone('vega', params.VEGA_VERSION)
+                                env.DOCKER_IMAGE_TAG_HASH = sh (returnStdout: true, script: "echo \"${repoVars.GIT_COMMIT}\"|cut -b1-8").trim()
```
Comment (daniel1302) [inline vars/pipelineDeploy.groovy:139]: We would also use the groovy command: 
`repoVars.GIT_COMMIT?.substring(0, 5))`

File: vars/pr.groovy [hunk 2]
```diff
@@ -110,3 +110,37 @@ Map injectPRParams() {
     }
     return params
 }
+
+List<String> getAllLabelsFor(Map config = [:]) {
```
Comment (daniel1302) [inline vars/pr.groovy:114]: Where do We use those functions? :)

**Review by daniel1302 — APPROVED** [review]

File: vars/pipelineDeploy.groovy (hunk 1 above)
Comment (daniel1302) [reply vars/pipelineDeploy.groovy:139]: But I like your solution as well

File: vars/pr.groovy (hunk 2 above)
Comment (fkondej) [reply vars/pr.groovy:114]: we will use them in vega core Jenkinsfile - possibly by adding `devnet` label, the pipeline will trigger Devnet CD for that PR.

File: vars/pipelineDeploy.groovy (hunk 1 above)
Comment (fkondej) [reply vars/pipelineDeploy.groovy]: done

**Pull request merged**